.github/workflows/tests.yml

```
@@ -0,0 +1,27 @@
+name: Python tests
+
+on:
+  push:
+    branches: 'master'
+  pull_request:
+    branches: 'master'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

I think it's nicer to use multiline syntax for these. Just like in the steps:
```suggestion
        python-version:
        - 3.8
        - 3.9
        - >-
          3.10
```
_(`3.10` is a special case, it's parsed as `3.1`, a float, if you don't explicitly use a string syntax)_